src/interfaces/EditInterface/useSaveInterface.js

```diff
@@ -0,0 +1,66 @@
+import { useShowCallout } from '@folio/stripes-acq-components';
+
+export default function useSaveInterface() {
+  const showCallout = useShowCallout();
+
+  return (
+    { vendorInterface, interfaceCredentials, interfaceId, interfaceOrg },
+    { username, password, ...values },
+    creds,
+    org,
+  ) => {
```

Comment:
Basically, the hook form is for just mixing-in the `showCallout`?
I would leave the `saveInterface()` be a standalone function, just add a `showMessage` parameter.
This way concerns are more separated.
Imagine we might want to save the things with no callouts.

Reply:
changed back to the fn